Tee'sApp/Controllers/ViewControllerB.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  FirstViewController.swift
+//  Tee'sApp
+//
+//  Created by La Pommeray, Thalia on 10/27/22.
+//
+
+import UIKit
+
+class ViewControllerB: UIViewController {
+    
+    var info: String?;
+    let manager = InfoViewModel()
+    var userData:[User] = []
+    var user = User.dummyUser()
+    var userIndex = 0
+    var infoForAlert:String = ""
```

Comment:
All of these can probably be `private`. (A good rule of thumb is to always default to `private`, and allow more access as you need)